### PR TITLE
fix(storybook): nuke dev-novel

### PR DESCRIPTION
**Pull request by tkrugg**
this PR removes dev-novel
closes: https://github.com/algolia/angular-instantsearch/issues/226

you check it out at this link: https://deploy-preview-410--angular-instantsearch.netlify.com/examples.html

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ cd angular-instantsearch
 
 ### Run
 
-You can watch & reload library using `> yarn dev` and then open http://localhost:3000 to access stories of Angular InstantSearch components rendered into [dev-novel](https://github.com/algolia/dev-novel).
+You can access stories of Angular InstantSearch component by running the storybook with `yarn example:storybook`
```
Comment (francoischalifour) [inline CONTRIBUTING.md]:
```suggestion
You can access stories of Angular InstantSearch components by running the storybook with `yarn example:storybook`.
```

File: scripts/build.sh [hunk 2]
```diff
@@ -16,6 +16,3 @@ mv dist/bundles/reset-min.css dist/bundles/instantsearch.min.css
 
 mv dist/bundles/algolia.css dist/bundles/instantsearch-theme-algolia.css
 mv dist/bundles/algolia-min.css dist/bundles/instantsearch-theme-algolia.min.css
-
-# build dev-novel
-MODE=build webpack --config webpack.demo.js
```
Comment (francoischalifour) [inline scripts/build.sh]: What is this replaced with?

File: scripts/netlify.sh [hunk 3]
```diff
@@ -15,9 +15,6 @@ yarn examples:router:build
 yarn examples:media:build
 yarn examples:storybook:build
 
-# build dev-novel
-MODE=build webpack --config webpack.demo.js
```
Comment (francoischalifour) [inline scripts/netlify.sh]: What is this replaced with?

Comment (tkrugg) [reply scripts/netlify.sh]: yarn examples:storybook:build

File: scripts/build.sh (hunk 2 above)
Comment (tkrugg) [reply scripts/build.sh]: it's redundant so I removed it

File: scripts/netlify.sh (hunk 3 above)
Comment (francoischalifour) [reply scripts/netlify.sh]: Ah yeah, I didn't see the change was already done!

**Pull request merged**